tests/config/clvk.conf

```diff
@@ -0,0 +1,2 @@
+cache_dir=failed
+compiler_temp_dir=not/overwritten/
```

Comment:
We should have an option of each type in the test file so all parse functions are exercised.